src/workerd/server/alarm-scheduler.c++

```diff
@@ -197,7 +198,7 @@ kj::Promise<void> AlarmScheduler::makeAlarmTask(kj::Duration delay,
 
   auto retryInfo = co_await ([&]() -> kj::Promise<RetryInfo> {
     try {
-      co_return co_await runAlarm(actorRef, scheduledTime);
+      co_return co_await runAlarm(actorRef, scheduledTime, retryNumber);
```

Comment:
Why pass in `retryNumber` vs using `entry.value.countedRetry` here as well? To be clear, I'm ok with it, but is it just a matter of convenience or is there a more meaningful reason?